react-native-libraries.json

```diff
@@ -6532,13 +6537,18 @@
   },
   {
     "githubUrl": "https://github.com/wix/react-native-ui-lib",
-    "examples": ["https://github.com/wix/react-native-ui-lib/tree/master/demo"],
+    "examples": [
+      "https://github.com/wix/react-native-ui-lib/tree/master/demo",
+      "https://github.com/kanzitelli/rnn-starter",
+      "https://github.com/react-native-tvos/rnn-starter-tv"
```

Comment:
Usually we do not add starters in examples, it rather for interactive demos and project solely build for the showcase of certain package. 

I think that if we make a precedence here, in not that long time, we will end up with list of dozen of entries for the most popular packages, which could impact the quality of example information. U

From the maintainer POV it might be hard to keep up with detailed reviews. Also the starter packages usually tends to have short life/maintenance span, and it might be difficult to keep the list updated and correct after few months.

Reply:
This particular starter app is officially called out by Wix as a good place to start to use RNN:

https://wix.github.io/react-native-navigation/docs/showcases#starter-kits

The TV fork will be the official testing app for RNN on TV and be maintained by me :)

Reply:
I'm more afraid about quality in general and it's degradation in time, not in particular quality of those packages.

Also, in the link I can see 4 starters, why you have added only one of them? What were the criteria here and why?
Accusations of favoritism or questions like that is also what I would like to avoid. 🙂

I'm not also perceive the linking repository in docs as a measurement of its quality, as a direct endorsement nor as making it "official" by that. 😉

Reply:
Ok I'll close this PR for now and see if I can come up with an example that isn't too closely derived from a particular starter app.  I'll take a look at the examples that are showing for other packages as guidance.

Reply:
I think we can merge this now, but just without the starters links. The `web` platform flags and the other example are great additions! :)